lib/bundler_api/versions_file.rb

```diff
@@ -0,0 +1,81 @@
+require 'bundler_api'
+require 'bundler_api/cdn'
+
+class BundlerApi::VersionsFile
+  PATH = "versions.list"
+  def initialize(connection)
+    @conn = connection
+  end
+
+  def create_or_update
+    if File.exists? PATH
+      update
+    else
+      create
+    end
+  end
+
+  def create
+    content = Time.now.to_i.to_s
+    content += "\n---\n"
+    content += gems_for_new_file
+
+    File.open(PATH, 'w') do |io|
+      io.write content
+    end
+
+    BundlerApi::Cdn.purge_versions_list
+  end
+
+  def update
+    to_write = with_new_gems
+    File.open(PATH, 'w') do |io|
+      io.write to_write
+    end
+
+    BundlerApi::Cdn.purge_versions_list
```

Comment:
Same here. We will want to purge fastly when new gems are added to the database, but not when this file is created or updated locally and then checked in.